01-node-ecosystem/lab/__test__/lib/greet.test.js

```diff
@@ -0,0 +1,24 @@
+'use strict'
+
+const hello = require('../../lib/greet.js');
+
+describe('Testing the hello module', () => {
+  it('should add hello to the passed string when the function is invoked with a parameter', () => {
+    expect(hello("world")).toBe('hello world');
```

Comment:
make sure to be consistent with single quotes

For example, 
```suggestion
    expect(hello('world')).toBe('hello world');
```